lighthouse-core/audits/metrics/total-blocking-time.js

```diff
@@ -27,23 +27,34 @@ class TotalBlockingTime extends Audit {
       title: str_(i18n.UIStrings.totalBlockingTimeMetric),
       description: str_(UIStrings.description),
       scoreDisplayMode: Audit.SCORING_MODES.NUMERIC,
-      requiredArtifacts: ['traces', 'devtoolsLogs'],
+      requiredArtifacts: ['traces', 'devtoolsLogs', 'TestedAsMobileDevice'],
     };
   }
 
   /**
-   * @return {LH.Audit.ScoreOptions}
+   * @return {{mobile: {scoring: LH.Audit.ScoreOptions}, desktop: {scoring: LH.Audit.ScoreOptions}}}
    */
   static get defaultOptions() {
     return {
-      // According to a cluster telemetry run over top 10k sites on mobile, 5th percentile was 0ms,
-      // 25th percentile was 270ms and median was 895ms. These numbers include 404 pages. Picking
-      // thresholds according to our 25/75-th rule will be quite harsh scoring (a single 350ms task)
-      // after FCP will yield a score of .5. The following coefficients are semi-arbitrarily picked
-      // to give 600ms jank a score of .5 and 100ms jank a score of .999. We can tweak these numbers
-      // in the future. See https://www.desmos.com/calculator/bbsv8fedg5
-      median: 600,
-      p10: 287,
+      mobile: {
+        // According to a cluster telemetry run over top 10k sites on mobile, 5th percentile was 0ms,
+        // 25th percentile was 270ms and median was 895ms. These numbers include 404 pages. Picking
+        // thresholds according to our 25/75-th rule will be quite harsh scoring (a single 350ms task)
+        // after FCP will yield a score of .5. The following coefficients are semi-arbitrarily picked
+        // to give 600ms jank a score of .5 and 100ms jank a score of .999. We can tweak these numbers
+        // in the future. See https://www.desmos.com/calculator/bbsv8fedg5
+        scoring: {
+          p10: 287,
+          median: 600,
+        },
+      },
+      desktop: {
+        // SELECT QUANTILES(tbt, 40), QUANTILES(tbt, 60) FROM [httparchive.pages.2020_04_01_desktop]
```

Comment:
ah wait a second, are you trying to say we should use 40th and 60th percentiles? I don't see `tbt` as a property either though :/

Reply:
> ah wait a second, are you trying to say we should use 40th and 60th percentiles? I don't see tbt as a property either though :/

this is `$._TotalBlockingTime`, but our mobile control points are way off from our standard method of choosing control points (see Deep's original comment above about the mobile control points).

Our normal process would give `{p10: 0, median: 83}` for the desktop curve. We *could* do that but that seems way too harsh, so I chose points on the desktop curve adjusted somewhat similarly to the existing mobile ones.

Happy to change if you feel differently, though.

Reply:
I wasn't asking because of the specific quantiles chosen :)

> I forgot that's actually the legacy SQL function and thought it was just a shorthand for APPROX_QUANTILES and an offset

Is the single sentence that explains 70% of my confusion. You *were* trying to use quantiles to return a single offset so 40th and 60th was the goal here rather than "give me 40 quantiles and then 60 quantiles". It all makes sense now!

Reply:
> I wasn't asking because of the specific quantiles chosen :)

ha, ok, I see now too :) Yeah, I'll update to make it actually clear